src/configLoaders/ReactConfigLoader.ts

```diff
@@ -0,0 +1,31 @@
+import ConfigLoader from './ConfigLoader';
+import createReactJestConfig from '../utils/createReactJestConfig';
+import * as path from 'path';
+
+export default class ReactConfigLoader implements ConfigLoader {
```

Comment:
Can you add some documentation: what does this class do, and why does it do so? I can't figure out why it exists.

Reply:
The `ConfigEditor` interacts with an implementation of `ConfigLoader` to load the configuration for a certain type of project specified in the `stryker.conf.js` ([see readme#configuration](https://github.com/stryker-mutator/stryker-jest-runner/blob/feat-new-jest-runner/README.MD#configuration)).

Personally I am not a big fan of writing comments in code because comments almost always lie. What would be a good place to document this?

Reply:
Personally, I prefer source code comments over people having to reverse-engineer it. Having them next to the code makes it easier to update it as soon as the code changes, while having it in a separate file makes that a bit harder.